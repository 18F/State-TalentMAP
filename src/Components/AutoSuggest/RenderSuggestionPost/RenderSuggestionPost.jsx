import React from 'react';
import PropTypes from 'prop-types';

const RenderSuggestionPost = ({ suggestion }) => (
  <div className="render-suggestion render-suggestion--post">
    {suggestion.location}
  </div>
);

RenderSuggestionPost.propTypes = {
  suggestion: PropTypes.shape({
    location: PropTypes.string.isRequired,
  }).isRequired,
};

export default RenderSuggestionPost;
