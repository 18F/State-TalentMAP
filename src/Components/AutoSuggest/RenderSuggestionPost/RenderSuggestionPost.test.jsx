import { shallow } from 'enzyme';
import React from 'react';
import toJSON from 'enzyme-to-json';
import RenderSuggestionPost from './RenderSuggestionPost';

describe('RenderSuggestionPostComponent', () => {
  const suggestion = {
    location: 'test',
  };
  it('is defined', () => {
    const wrapper = shallow(
      <RenderSuggestionPost
        suggestion={suggestion}
      />,
    );
    expect(wrapper).toBeDefined();
  });

  it('matches snapshot', () => {
    const wrapper = shallow(
      <RenderSuggestionPost
        suggestion={suggestion}
      />,
    );
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
