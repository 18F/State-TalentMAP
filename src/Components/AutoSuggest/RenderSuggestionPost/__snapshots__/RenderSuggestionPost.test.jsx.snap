// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RenderSuggestionPostComponent matches snapshot 1`] = `
<div
  className="render-suggestion render-suggestion--post"
>
  test
</div>
`;
